spyder/widgets/findinfiles.py

```diff
@@ -596,7 +637,8 @@ def append_result(self, results, num_matches):
         filename, lineno, colno, line = results
 
         if filename not in self.files:
-            item = FileMatchItem(self, filename)
+            item = FileMatchItem(self, filename, self.sorting_status)

```

Comment:
I disable sorting only on file entries and not line entries, to accomplish that I use the dict